schedule/yast/sle/flows/default_sle15sp3_aarch64.yaml

```diff
@@ -1,37 +1,48 @@
 ---
-name: qam-yast_self_update
-description: installation using self_update as boot parameter
-vars:
-  YUI_REST_API: 1
-schedule:
+# Default ordered sequence of steps for sle15sp3 with maintenance updates
+bootloader:
   - installation/bootloader_start
+setup_libyui:
   - installation/setup_libyui
+product_selection:
   - installation/product_selection/install_SLES
-  - installation/validate_self_update
+license_agreement:
   - installation/licensing/accept_license
+registration:
   - installation/registration/register_via_scc
+extension_module_selection:
   - installation/module_registration/register_extensions_and_modules
-  - installation/module_registration/add_additional_regcodes
-  - installation/module_registration/import_untrusted_gnpupg_key
+add_on_product:
   - installation/add_on_product/add_maintenance_repos
```

Comment:
our default is the one with dud, not this one.

Reply:
My aarch64 and s390x cases don't use dud.  At default, should I still need change to use dud?

Reply:
my bad, we only did that for x86_64, we can develop that in the future, it is fine then, thanks.